### PR TITLE
Description Updates

### PR DESCRIPTION
**Pull request by cheeeeeeeeeen**
This is just an improvement for `NewLangPickup` and `NewLangDesc` found on items.

e.g. Remove `(+0 damage per stack)` if the bonus is `0`. Display otherwise if it's greater than `0`.

### CODE REVIEW

File: Items/Eqp/Prescriptions.cs [hunk 1]
```diff
@@ -21,9 +21,9 @@ public class Prescriptions : Equipment<Prescriptions> {
         public float dmgBoost {get;private set;} = 10f;
 
         public BuffIndex prescriptionsBuff {get;private set;}
-        protected override string NewLangName(string langid = null) => displayName;        
-        protected override string NewLangPickup(string langid = null) => "Increase damage and attack speed for " + duration.ToString("N0") + " seconds.";        
-        protected override string NewLangDesc(string langid = null) => "While active, increases <style=cIsDamage>base damage by " + dmgBoost.ToString("N0") + " points</style> and <style=cIsDamage>attack speed by " + Pct(aSpdBoost) + "</style>. Lasts <style=cIsDamage>" + duration.ToString("N0") + " seconds</style>.";
+        protected override string NewLangName(string langid = null) => displayName;
+        protected override string NewLangPickup(string langid = null) => FormatNewLangPickup();
```
Comment (ThinkInvis) [inline Items/Eqp/Prescriptions.cs]: NewLangPickup/NewLangDesc are already methods; creating single new methods is redundant in this case. `type method(args) => retv;` is a shorthand for `type method(args) {return retv;}`.

File: Items/EqpLunar/Lantern.cs [hunk 2]
```diff
@@ -66,7 +66,23 @@ public class Lantern : Equipment<Lantern> {
...
+        {
+			string desc = "Drop a lantern that fears";
+			if (damage > 0f) desc += " and damages";
+			desc += $" enemies for {duration:N0} seconds.";
+			return desc;
+        }
+
+		private string FormatNewLangDesc()
+		{
+			string desc = $"Sets a {range:N0}m, {duration:N0}-second AoE which <style=cIsUtility>fears enemies</style>";
```
Comment (ThinkInvis) [inline Items/EqpLunar/Lantern.cs]: "###unit" is never grammatically correct. "### unit" usually is, but "###-nonAbbreviatedUnit" is preferred in some cases like the above.

File: Items/T1/BitterRoot.cs [hunk 3]
```diff
@@ -20,8 +20,8 @@ public class BitterRoot : Item<BitterRoot> {
         public float healthCap {get; private set;} = 3f;
 
         protected override string NewLangName(string langid = null) => displayName;        
-        protected override string NewLangPickup(string langid = null) => "Gain " + Pct(healthMult) + " max hp.";
-        protected override string NewLangDesc(string langid = null) => "Increases <style=cIsHealing>health</style> by <style=cIsHealing>" + Pct(healthMult) + "</style> <style=cStack>(+" +Pct(healthMult)+ " per stack, linear)</style>, up to a <style=cIsHealing>maximum</style> of <style=cIsHealing>+"+Pct(healthCap)+"</style>.";        
+        protected override string NewLangPickup(string langid = null) => "Gain " + Pct(healthMult) + " max health.";
```
Comment (ThinkInvis) [inline Items/T1/BitterRoot.cs]: Use of "hp" vs "health" here is a stylistic choice for the pickup text, as the former matches with the RoR1 item's pickup text.

File: Items/TLunar/OldBox.cs [hunk 4]
```diff
@@ -27,7 +27,7 @@ public class OldBox : Item<OldBox> {
...
         protected override string NewLangPickup(string langid = null) => "Chance to fear enemies when attacked.";
-        protected override string NewLangDesc(string langid = null) => "<style=cDeath>When hit for more than " + Pct(healthThreshold) + " max health</style> <style=cStack>(/2 per stack)</style>, <style=cIsUtility>fear enemies</style> within <style=cIsUtility>" + radius.ToString("N0") + " m</style> for <style=cIsUtility>" + duration.ToString("N1") + " seconds</style>. <style=cIsUtility>Feared enemies will run out of melee</style>, <style=cDeath>but that won't stop them from shooting you.</style>";
+        protected override string NewLangDesc(string langid = null) => "<style=cDeath>When hit for more than " + Pct(healthThreshold) + " max health</style> <style=cStack>(/2 per stack)</style>, <style=cIsUtility>fear enemies</style> within <style=cIsUtility>" + radius.ToString("N0") + " m</style> for <style=cIsUtility>" + duration.ToString("N1") + " seconds</style>. <style=cIsUtility>Feared enemies will run away</style>, <style=cDeath>but that won't stop them from performing ranged attacks</style>.";
```
Comment (ThinkInvis) [inline Items/TLunar/OldBox.cs]: Word choice here was intentional to clearly emphasize "this is a bad idea for melee characters".

**Review by ThinkInvis — CHANGES_REQUESTED** [review]

File: Items/T1/BitterRoot.cs (hunk 3 above)
Comment (cheeeeeeeeeen) [reply Items/T1/BitterRoot.cs]: Can we at least capitalize it from `hp` to `HP`? Still your call though.

File: Items/TLunar/OldBox.cs (hunk 4 above)
Comment (cheeeeeeeeeen) [reply Items/TLunar/OldBox.cs]: Noted.

File: Items/EqpLunar/Lantern.cs (hunk 2 above)
Comment (cheeeeeeeeeen) [reply Items/EqpLunar/Lantern.cs]: Noted.

File: Items/Eqp/Prescriptions.cs (hunk 1 above)
Comment (cheeeeeeeeeen) [reply Items/Eqp/Prescriptions.cs]: Ah, so you mean I'll just directly set the the format there? Alright noted.

**Pull request merged**